source/includes/_enrichment.md

```diff
@@ -435,8 +431,7 @@ first_name (optional) | The person's first name | Tim
 last_name (optional) | The person's last name | Zheng
 name (optional) | The person's full name | Tim Zheng
 email (optional) | The person's email | example@domain.com
-email_md5 (optional) | The person's hashed email | 8d935115b9ff4489f2d1f9249503cadf
-email_sha256 (optional) | The person's sha 256 hashed email | 97817c0c49994eb500ad0a5e7e2d8aed 51977b26424d508f66e4e8887746a152
+hashed_email (optional) | The person's md5 or sha256 hashed email | 8d935115b9ff4489f2d1f9249503cadf OR 97817c0c49994eb500ad0a5e7e2d8aed 51977b26424d508f66e4e8887746a152
```

Comment:
The gap in between` 97817c0c49994eb500ad0a5e7e2d8aed 51977b26424d508f66e4e8887746a152` is to for new line?

Reply:
Yes its for the new line..